src/scripts/tag_replacer.js

```
@@ -137,7 +137,8 @@ const replaceTag = async ({ uuid, oldTag, newTag, appendOnly }) => {
   while (resource) {
     await Promise.all([
       apiFetch(resource).then(({ response }) => {
-        taggedPosts.push(...response.posts);
+        const posts = response.posts.filter(object => object.objectType === 'post');
```

```suggestion
        const posts = response.posts.filter(({ canEdit }) => canEdit === true);
```

---

Oh yeah, I guess this makes sense for mass deleter/mass unliker as well; in all cases we are collecting posts to edit them, so non-editable entries of any sort would be bad.

Come to think of it... I feel like I would want a warning instead of a silent failure if I tried to tag replace a blazed post hmmmmm

---

Mass Deleter might need it for group blog purposes, I actually can't remember if non-admin group members can access a group blog's drafts at all, or if draft editing is free for any group member, or not.

But liking a post doesn't create an editable copy... unliking does require more than a post ID though, so filtering to objects with the necessary data to send an unlike request would be good.

---

Oh, right. There is a `canLike` field, though I don't know what toggles it, and thus whether it helps with any edge cases (I guess it would be about the user-has-blocked-you case?)

I guess re: "warning instead of silent failure" maybe the best way to do that would be to remove ads, then filter a second time and display "found x posts (y uneditable)." Or do you think that's showing how the sausage is made too much, so to speak, and it should just be a silent failure?

---

You can always unlike posts regardless of blocks. If a blog has you blocked, you can't like their posts but can unlike them.

I don't see that there's a clear benefit to displaying the count of uneditable posts.., it's already an edge-case that they will occur at all, much simpler to just filter them out silently. We can always follow up and implement this later if anyone actually notices.